modules/vba_documents/README.yml

```diff
@@ -292,50 +325,85 @@ components:
         document package (unreadable PDF, etc) may cause the status to change to “error”.
 
       required:
+        - id
+        - type
+        - attributes
         - guid
         - status
         - code
         - message
         - detail
       properties:
-        guid:
-          description: The document upload identifier.
+        id:
```

Comment:
minor irritation about inconsistent use of `id` vs `guid`

Reply:
Yeah...I should have thought this through better, with JSON API there's no need to have guid represented in the object attributes proper :( But I'm guessing it's being consumed at this point, so don't want to remove it. I could change the bulk report operation request object key to `ids` with a view towards deprecating the guids attribute and standardizing on 'ids'.